.rubocop.yml

```diff
@@ -1,5 +1,7 @@
 # Please keep AllCops, Bundler, Style, Metrics groups and then order cops
 # alphabetically
+AllCops:
+  TargetRubyVersion: 2.4
 inherit_from:
   - https://raw.githubusercontent.com/hanami/devtools/master/.rubocop.yml
```

Comment:
@jodosha should we update this `inherit_from` key with `.rubocop_unstable.yml`?

Reply:
@AlfonsoUceda Yes, and `TargetRubyVersion` should be updated in `.rubocop_unstable.yml`.

Reply:
:+1: